gufe/components/smallmoleculecomponent.py

```diff
@@ -79,6 +81,10 @@ class SmallMoleculeComponent(Component, Serializable):
     def __init__(self, rdkit: RDKitMol, name: str = ""):
         name = _ensure_ofe_name(rdkit, name)
         _ensure_ofe_version(rdkit)
+        if not list(rdkit.GetConformers()):
+            # do we require at least 1 or exactly 1?
```

Comment:
I think we can allow more than one, but raise a warning that only the 0th element will be used as the "real" conformation.

Reply:
warning added